.github/workflows/build-test-manylinux-aarch64.yml

```diff
@@ -0,0 +1,45 @@
+# AUTOGENERATED FROM build-test-manylinux-aarch64.jsonnet DO NOT MODIFY
+name: build-test-manylinux-aarch64
+"on":
+  workflow_dispatch:
+  workflow_call:
+jobs:
+  build-wheels:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - uses: docker/setup-qemu-action@v3
+      - uses: docker/setup-buildx-action@v2
+      - uses: depot/setup-action@v1
+      - name: Build and test python wheel
+        id: build-semgrep-wheel
+        uses: depot/build-push-action@v1.9.0
+        with:
+          project: fhmxj6w9z8
+          platforms: linux/arm64
+          outputs: type=docker,dest=/tmp/image.tar
+          target: semgrep-wheel
+          buildx-fallback: true
+      - name: Extract wheel from docker image
+        run:
+          "\n        # load the docker image containing the semgrep python wheel\n
+          \       docker load --input /tmp/image.tar\n\n        # create a new docker
+          container using the image we just loaded\n        # note: `docker create`
+          simply prepares the container filesystem -- nothing is executed!\n        CONTAINER_ID=$(docker
+          create ${{ steps.build-semgrep-wheel.outputs.imageid }})\n\n        # use
+          `docker export` to extract the python wheel zipfile out of the container\n
+          \       docker export $CONTAINER_ID | tar xv semgrep/cli/dist.zip\n\n        #
+          clean up after ourselves\n        docker rm $CONTAINER_ID\n\n        # note:
+          this was originally accomplished by running `cat` from within the container
+          and redirecting stdout to a file:\n        #\n        # docker run --platform
+          linux/arm64 --rm ${{ steps.build-semgrep-wheel.outputs.imageid }} cat cli/dist.zip
+          > /tmp/dist.zip\n        #\n        # we ended up hitting an edge case where
+          the final ~100kb doesn't always get written to disk, which results in a corrupted
+          zip file.\n        # our theory is that the pipe between the container and
+          the host is being closed prematurely.\n      "
```

Comment:
This looks wrong.

Reply:
I guess it's correct but the quoting and `\n` escapes make it unreadable. And it probably doesn't matter if it's not meant to be human-readable.

Reply:
yes it's because I use the Yaml OCaml library to generate those yaml and it does not respect the ||| format for long string that I use in the .jsonnet files. Maybe there is a pretty printing option to use the : |   multiline option of YAML for those long strings.